src/public/utils.ts

```diff
@@ -38,7 +38,8 @@ function stopWaiting() {
  * Go back to linkurious home page
  */
 function goToLinkurious(sourceKey: string) {
-  const location = window.location.origin + `/dashboard?key=${sourceKey}`;
+  const basePath = window.location.pathname.split('/')[1];
```

Comment:
I would make sure the the basePath is actually defined correctly before start using it and not do "this basePath === 'plugins' ? '' : basePath" on the line where you define the location.

Also avoid the situation where basePath is undefined and you redirect to "//dashboard"